app/src/main/java/ch/epfl/sdp/blindly/helpers/Message.kt

```diff
@@ -10,17 +15,16 @@ class Message {
     // timestamp is to achieve correct ordering of the messages and ensure fetching is in
     // same ordering the next time users open the chat
     var timestamp: Long? = null
-    var messageText: String? = null
+    var messageText: T? = null
```

Comment:
Is there a reason why the message contains a generic type now ? I guess it allows us to do more things with it, but since the message will only contain a String, is it really important to do so ?

Reply:
See my comment on @cmsigrist 's remark https://github.com/BlindlyTeam/Blindly/pull/182#discussion_r627555824

Reply:
No it won't the LiveLocation database sends a BlindlyLatLng.